packages/devtools/web/index.html

```diff
@@ -24,6 +25,18 @@
     <link rel="stylesheet" href="packages/devtools/src/logging/logging.css">
     <link rel="stylesheet" href="packages/devtools/src/timeline/timeline.css">
 
+    <script type="text/javascript">
+        (function() {
```

Comment:
I added this code here to make sure the dark theme is loaded at initial page load
if `theme=dark" is present.
Otherwise you would get a flash of the light theme which would look bad.
I can clean this up to not use document.write
as I should feel bad for using it. :)